### PR TITLE
react to removal of PlatformAbstractions

### PR DESCRIPTION
**Pull request by anurse**
This PR removes all references to PlatformAbstractions.

Part of aspnet/PlatformAbstractions#50

### CODE REVIEW

File: test/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest.csproj [hunk 1]
```diff
@@ -13,7 +13,6 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging.Testing" Version="$(AspNetCoreVersion)" />
-    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="$(AspNetCoreVersion)" />
```
Comment (moozzyk) [inline test/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest.csproj]: Weren't we use it or the functionality was moved to one of the packages we are already using?

**Comment by moozzyk** [comment]
LGTM

File: test/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest.csproj (hunk 1 above)
Comment (anurse) [inline test/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest/Microsoft.AspNetCore.WebSockets.Internal.ConformanceTest.csproj]: Nope, never used it :). Many of the others repos had the same thing going on. Old code that was removed and the reference never updated.

**Comment by anurse** [comment]
CI failures seem unrelated. Rebased and trying again.

**Pull request merged**